dynamixel_controllers:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 10

  velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: first

  position_controller:
    type: position_controllers/JointPositionController
    joint: second

dynamixel_control_hw:
  serial_interface: /dev/ttyUSB0
  loop_frequency: 50
  cycle_time_error_threshold: 0.1
  baudrate: 1000000 # in bauds
  read_timeout: 0.05 # in seconds
  scan_timeout: 0.05 # in seconds
  # Cofniguration of the servos
  servos:
    first:
      # hardware ID of the actuator
      id: 1
      # offset to be added, in radians, to the position of an actuator
      offset: 4.5
      command_interface: velocity
    second:
      id: 26
      # max joint speed, in rad/s
      # it shoud NOT be used for actuators in velocity mode
      max-speed: 4
  # default mode
  default_command_interface: position

